backend/.env.example

```
@@ -26,6 +28,7 @@ DJANGO_SLACK_CLIENT_ID=
 DJANGO_SLACK_CLIENT_SECRET=
 DJANGO_SLACK_SEARCH_TOKEN=
 DJANGO_SLACK_SIGNING_SECRET=
+SLACK_BOT_TOKEN_<WORKSPACE_ID>=
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The SLACK_BOT_TOKEN_<WORKSPACE_ID> key has incorrect delimiter

(IncorrectDelimiter)